src/mobile-app/components/sensor.tsx

```diff
@@ -138,7 +139,7 @@ export const SensorComponent: React.FC<IProps> = ({sensor, onResetAll, onSetMode
       </div>
     );
   };
-  const connectionClassName = `${css.connection} ${error ? css.error : (connected ? css.connected : css.disconnected)}`;
+  const connectionClassName = `${css.connection} ${error ? css.error : ""}`;
```

Comment:
This change removes the background color for the row when connected / disconnected (and also on error).  I ran this branch locally and this is what it looks like in the sensor demo:

![image](https://user-images.githubusercontent.com/112938/74158336-e6354200-4be7-11ea-8a51-99f3f8362255.png)